src/main/java/org/fogbowcloud/saps/engine/core/database/ImageDataStore.java

```diff
@@ -25,7 +25,8 @@
 	public String DATASTORE_PORT = "datastore_port";
 
 	SapsImage addImageTask(String taskId, String dataset, String region, Date date, int priority, String user,
-			String inputGathering, String inputPreprocessing, String algorithmExecution) throws SQLException;
+			String inputdownloadingPhaseTag, String digestInputdownloading, String preprocessingPhaseTag,
```

Comment:
your modifications look good to me. However, I think you should take the opportunity to refactor the code (you can choose to modify in this PR or to open another one, with refactor-only modifications).

1. delete all unused methods from this interface
2. This `ImageDataStore` is our `Catalog` right? If so, change the name.
3. do we need two `addImageTask` method? Maybe, we can keep the one that receives `SapsImage` parameter

Reply:
also, there is no need to use `public` in the fields declared in an interface

Reply:
I created an issue (#73 ) reporting on.